xpk.py

```diff
@@ -3862,12 +3870,43 @@ def get_pathways_rm_args(args) -> str:
               - --pathways_persistent_compilation_cache=false
               - --pathways_compilation_mode=compile_at_worker
               - --pathways_tmp_dir_pattern={args.pathways_gcs_location}
-              - --pathways_resource_manager_expected_num_worker_jobs={args.num_slices}"""
+              - --pathways_expected_instances={expected_instances}"""
```

Comment:
Thinking ahead, in terms of exposure is there a need to add this argument (and relevant logic) in XPK? 

Would it be better to have this argument as part of the RM image?

Reply:
Right now this is how Pathways takes in the argument, and it likely won't be in the roadmap to put this logic in the near future to support various accelerator options (possibly shapes and sizes and number).